django/contrib/admin/models.py

```diff
@@ -44,6 +44,27 @@ def log_action(
             change_message=change_message,
         )
 
+    def log_actions(self, user_id, queryset, action_flag, change_message=""):
+        """
+        Method to bulk_create multiple LogEntry objects
+        from a given queryset.
```

Comment:
This docstring is wrapping a little early; the line length for comments & docstrings is 79 chars.